slide-in-with-cta/template.css

```
@@ -0,0 +1,204 @@
+/* ==========================================================================
+   Slide-In
+========================================================================== */
+
+#evg-slide-in-with-cta {
+    position: absolute;
+    z-index: 100000;
+    top: auto !important;
+    right: 10%;
```

```suggestion
    right: 5%;
```

right: 10% positions the template too close to the center. Let's go with 5% instead